docs/references/express/overview.mdx

```diff
@@ -0,0 +1,108 @@
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```sh filename=".env.local"
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+</Steps>
+
+## Available methods
+
+### `clerkMiddleware()`
+
+The `clerkMiddleware()` helper integrates Clerk authentication into your Express application. It is required to be set in the middleware chain before using other Clerk utilities, such as `requireAuth` and `getAuth()`. It is re-exported from the `@clerk/nextjs/server` package, so see the [Next.js reference documentation](https://clerk.com/docs/references/nextjs/clerk-middleware) for more information.
+
+```js
+import { clerkMiddleware } from '@clerk/express'
+```
+
+### `requireAuth`
+
+The `requireAuth` helper protects routes in your Express application by checking if the user is authenticated or authorized. See the [reference documentation](/docs/references/express/require-auth) for more information.
+
+### `getAuth()`
+
+The `getAuth()` helper retrieves the authentication state, allowing you to protect your API routes or gather relevant data. It is re-exported from the `@clerk/nextjs/server` package, so see the [Next.js reference documentation](https://clerk.com/docs/references/nextjs/get-auth) for more information on how to use it.
+
+```js
+import { clerkMiddleware, getAuth } from '@clerk/express'
+import express from 'express'
+
+const app = express()
+
+const port = 3000
+
+// Apply centralized middleware
+app.use(clerkMiddleware())
+
+// Protect a route based on authorized
+hasPermission = (request, response, next) => {
+  const auth = getAuth(request)
+
+  // Handle if the user is not authorized
+  if (!auth.has({ permission: 'org:admin:testpermission' })) {
+    response.status(403).send('Forbidden')
+    return
+  }
+
+  return next()
+}
+
+router.get('/path', requireAuth, hasPermission, getHandler)
+
+// Start the server and listen on the specified port
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`)
+})
+```
+
+### `clerkClient`
+
+[Clerk's JavaScript Backend SDK](/docs/references/backend/overview) exposes Clerk's Backend API resources and low-level authentication utilities for JavaScript environments. For example, if you wanted to get a list of all users in your application, instead of creating a fetch to Clerk's `https://api.clerk.com/v1/users` endpoint, you can use the `users.getUserList()` method provided by the JavaScript Backend SDK.
+
+All resource operations are mounted as sub-APIs on the `clerkClient` object. See the [reference documentation](/docs/references/backend/overview#usage) for more information.
```

Comment:
We also need an example here of how to import the `clerkClient` object and run some simple operation using it